examples/todo/js/subscriptions/AddTodoSubscription.js

```diff
@@ -12,11 +12,8 @@ export default class AddTodoSubscription extends RelaySubscriptions.Subscription
   getSubscription() {
     return Relay.QL`subscription {
       addTodoSubscription {
-        clientMutationId
         todoEdge {
-          __typename
           node {
-            __typename
```

Comment:
when the babel plugin source code is back (https://github.com/facebook/relay/issues/1348) we can try to see if we can fix it there.

Just send another PR re-adding them for now :D